crates/ratchet-core/src/ops/softmax.rs

```diff
@@ -99,14 +103,16 @@ impl Softmax {
             ],
             device.compute_features().clone(),
         );
-        self.register_bindings::<P, T, N>(&mut kernel_builder, inplace)?;
+        self.register_bindings::<P>(&mut kernel_builder, inplace)?;
         kernel_builder.write_metadata::<SoftmaxMeta>();
 
-        let dt = T::DT;
+        let dt = P::T::DT;
         let accessor = P::render_type();
 
         let BLOCK_SIZE = workgroup_size.x.render();
-        let minFloat = T::from(-65500).unwrap().render();
+        let minFloat = <<P as WgslPrimitive>::T as NumCast>::from(-65500)
```

Comment:
There is probably a less verbose way of doing this.
All the float types could have the minimum value as an associated const (would not be surprised if that is already the case)

Reply:
@ivarflakstad They do have a minimum value, unfortunately the rust minimum value for rust is slightly lower than the minimum value for WGSL.

Reply:
Set your own as an associated const?

Reply:
```rust
trait WgslDType {
  const MIN: usize
}

impl WgslDType for f32 {
  const MIN: usize = 42
}
```
or something like that